examples/grpc/greeter/client.cpp

```diff
@@ -0,0 +1,84 @@
+#include "greeter.grpc.fb.h"
```

Comment:
Not sure why I overlooked the `samples` directory, maybe just a reflex since `examples` is more common. I'll move them into `grpc/samples`.